config/filter.d/http-badbots.conf

```diff
@@ -3,14 +3,25 @@
 # Regexp to catch known spambots and software alike. Please verify
 # that it is your intent to block IPs which were driven by
 # above mentioned bots.
+#
+# You can redefine or extend both parameters "badbots" and "badbotscustom" 
+# in your jails [apache-badbots] or [nginx-badbots]:
+#
+# - to redefine (totally overwrite) it using your own expression only:
+#     filter = http-badbots[badbots=BadBot0, badbotscustom=BadBot1|BadBot2]
+#
+# - to expand it with your own expression use <known/param> syntax:
+#     filter = http-badbots[badbots=BadBot0|<known/badbots>]
 
-
-[Definition]
+[Init]
```

Comment:
Why not if under circumstances the order of sections has no meaning? :)
And you theoretically agree with me that the [Init] should take place (in all filter) above the [Definition] (another PR).